src/components/Contact/styles.scss

```diff
@@ -0,0 +1,103 @@
+@import 'theme/colors.scss';
+@import 'theme/spacings.scss';
+@import 'theme/shadows.scss';
+
+.contact {
+  display: flex;
+  flex-flow: column nowrap;
+
+  width: 100%;
+  width: 100vw;
+
+  padding: $spc-l 0 $spc-xxl 0;
```

Comment:
Just a tip, this can be shorted as:
`padding: $spc-l 0 $spc-xxl;`

Reply:
Nice, thanks 😃